pkg/apis/core/validation/validation.go

```diff
@@ -3973,6 +4001,10 @@ func ValidatePodUpdate(newPod, oldPod *core.Pod, opts PodValidationOptions) fiel
 	// 1.  spec.containers[*].image
 	// 2.  spec.initContainers[*].image
 	// 3.  spec.activeDeadlineSeconds
+	// 4.  spec.containers[*].startupProbe.terminationGracePeriodSeconds
+	// 5.  spec.containers[*].livenessProbe.terminationGracePeriodSeconds
+	// 6.  spec.initContainers[*].startupProbe.terminationGracePeriodSeconds
+	// 7.  spec.initContainers[*].livenessProbe.terminationGracePeriodSeconds
 
```

Comment:
don't we need the same change for podSpec.TerminationGracePeriodSeconds (tolerate update from negative to 1)?